python/pyspark/testing/utils.py

```diff
@@ -39,6 +54,29 @@
     # No NumPy, but that's okay, we'll skip those tests
```

Comment:
unrelated, but better to change `except` to `except ImportError`

Reply:
Cool!